canonical-web-and-design/ubuntu.com
# Update links to insights.u.c to blog

#5013  Done

- Update redirects and links to insights to point to blog
- Uploaded one pdf to assets

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    - redirects.yaml
    - https://0.0.0.0:8001/desktop/developers
    - https://0.0.0.0:8001/desktop/partners
    - https://0.0.0.0:8001/financial-services/
    - https://0.0.0.0:8001/internet-of-things/appstore
    - https://0.0.0.0:8001/internet-of-things/gateways
    - https://0.0.0.0:8001/internet-of-things/robotics
    - https://0.0.0.0:8001/kubernetes/
    - https://0.0.0.0:8001/kubernetes/managed
    - https://0.0.0.0:8001/public-cloud/
    - https://0.0.0.0:8001/security/
    - https://0.0.0.0:8001/telecommunications/
    - https://0.0.0.0:8001/training/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the old links now work with the replaced links
